### DISCUSSION TITLE
Search on table and column name now available (Dagster+)

### DISCUSSION THREAD

**Discussion by clairelin135** [dagster-io/dagster#24424] [General]
We've landed improvements to the asset search experience to enable search on table name and column name. This feature is now available in experimental mode to Dagster+ users, under the "experimental column search" feature flag.

If you are seeing unexpected behavior with this feature, you can revert back to the prior search experience by toggling `User Settings` > `Experimental column search` to be off.

Please share any feedback below!

<img width="1188" alt="image" src="https://github.com/user-attachments/assets/44bd3064-5979-4d67-9a10-292542ddda10">

<img width="1455" alt="image" src="https://github.com/user-attachments/assets/45e44bc9-d49f-4d57-aa4a-2b99d655f97e">

<img width="1296" alt="Screenshot 2024-11-07 at 2 25 01 PM" src="https://github.com/user-attachments/assets/aa461e6f-b282-49e2-9924-8d7d2cc0dc9d">

**Comment by Elliot2718** [comment]
Something that would make the searching (and "views") more useful would to be able to filter by asset keys. E.g. we set our assets representing dbt models to be `<dbt project name>/<database>/<schema>/<model name>`. So it would be nice to be able to apply a filter to filter for only assets in one particular asset prefix, or a set of asset prefixes.

**Reply by clairelin135** [reply]
@Elliot2718 thanks for sharing, agree that would be useful! I'll add this to our backlog.
